pkg/apis/application/v1alpha1/types.go

```diff
@@ -1725,6 +1725,14 @@ type AppProjectSpec struct {
 	NamespaceResourceWhitelist []metav1.GroupKind `json:"namespaceResourceWhitelist,omitempty" protobuf:"bytes,9,opt,name=namespaceResourceWhitelist"`
 	// List of PGP key IDs that commits to be synced to must be signed with
 	SignatureKeys []SignatureKey `json:"signatureKeys,omitempty" protobuf:"bytes,10,opt,name=signatureKeys"`
+	// OrphanedResourceKeyWhitelist contains list of whitelisted orphaned resources
+	OrphanedResourceWhitelist []OrphanedResourceKey `json:"orphanedResourceWhitelist,omitempty" protobuf:"bytes,11,opt,name=orphanedResourceWhitelist"`
```

Comment:
Please add `OrphanedResourceWhitelist` field to `OrphanedResourcesMonitorSettings` data structure - it holds orphaned resources related settings.

I think `whiltelist/blacklist` terminology does not fit very well here, because we don't allow or prohibit anything here. Instead, I would suggest using `include/ignore` (add only ignore now and we can add include later ). Here is how sample project spec might look like:


```
spec:
  orphanedResources:
    warn: false
    ignore:
      - group: appv1
        kind: Deployment
        name: '*'
```

What do you think?